### PR TITLE
Added Docker to Vagrant VM init script

### PR DESCRIPTION
**Pull request by nicknezis**
Added Docker to the Vagrant init script. This is useful when needing to build docker images from within a Linux environment.

Also updated the NET_PREFIX to adhere to new Vagrant default range. The previous value would error when running `vagrant up`.

### CODE REVIEW

**Comment by nicknezis** [comment]
Fixes #3698

**Review by nwangtw — APPROVED** [review]

**Review by surahman — APPROVED** [review]
I have retriggered the builds.

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # vi: set ft=ruby :
 
 SECONDARIES=0
-NET_PREFIX="192.168.25."
+NET_PREFIX="192.168.56."
```
[the remark this answers is not in the document]
Comment (nicknezis) [reply vagrant/Vagrantfile:19]: When I tried to run in Vagrant, I got an error message about it not being in the acceptable subnet range. I think this is a change in Vagrant, so I updated to fit into the new preferred range.

[the remark this answers is not in the document]
Comment (surahman) [reply vagrant/Vagrantfile:19]: Got it, sorry lost sight of the last paragraph in the PR'S initial comment.

**Review by joshfischer1108 — APPROVED** [review]

**Pull request merged**